src/components/Header/Header.js

```diff
@@ -1,20 +1,19 @@
 import React from 'react';
 import { Nav, Navbar } from 'react-bootstrap';
 import './Header.css'
-import { constants } from '../../constants';
 
 function Header() {
     return (
         <Navbar expand="lg" className='navbar-bg' >
-          <Navbar.Brand className='navbar-brand-style'>{constants.REACT_APP_NAV_BRAND}</Navbar.Brand>
+          <Navbar.Brand className='navbar-brand-style'>{process.env.REACT_APP_NAV_BRAND}</Navbar.Brand>
```

Comment:
constants in process.env has been moved to constants.js so change all process.env. to constants proces.env.